docs/en/infraops/infra-ui-intro.asciidoc

```diff
@@ -0,0 +1,73 @@
+[[infrastructure-ui-overview]]
+[role="xpack"]
+== {infra-ui} UI
+
+beta[]
+
+After you have infrastructure monitoring <<install-infrastructure-monitoring,up and running>>
+and data is streaming to {es}, use the infrastructure monitoring UI in {kib} to monitor your
+infrastructure and identify problems in real time.
+
+// REVIEWERS: I'm not trying to describe how the UI works;
+// I want to provide enough info to generate interest in the UI and help users
+// understand what is possible. The Kibana docs will describe how to do stuff
+// in the UI, but I think users need to see image here to understand the value
+// and power of the UI. Screen captures with realisitc data (to be added) will
+// help users understand the value.
+
+[float]
+=== Monitor your hosts and containers
+
+You start with an overview of the hosts and containers running in your
+infrastructure. The overview provides a summary of high-level metrics, like CPU
+usage, system load, memory usage, and network traffic, to help you assess the
+overall health of your systems and services.
```

Comment:
Maybe add a note somewhere that the view can be (and has to be) switched between hosts, containers and (k8s) pods by using the links at the top right.

Reply:
I'd like to keep this discussion more conceptual. We'll be adding other topics to Kibana to cover the functioning of the UI. Thanks for your input!